controlplane/kubeadm/internal/webhooks/kubeadm_control_plane.go

```diff
@@ -44,6 +44,8 @@ import (
 	"sigs.k8s.io/cluster-api/util/version"
 )
 
```

Comment:
I've opened a PR against your PR with a suggestion. Would have been tricky to put this in a review comment :)

(xref: https://github.com/Karthik-K-N/cluster-api/pull/266)